test/commands/generate/migration_test.rb

```diff
@@ -0,0 +1,41 @@
+require 'test_helper'
+require 'lotus/commands/generate/migration'
+
+describe Lotus::Commands::Generate::Migration do
+  let(:opts)    { Hash.new }
+  let(:env)     { Lotus::Environment.new(opts) }
+  let(:command) { Lotus::Commands::Generate::Migration.new(migration_name, env, cli) }
+  let(:cli)     { Lotus::Generate.new }
+
+  def create_temporary_dir
+    tmp = Pathname.new(@pwd = Dir.pwd).join('tmp/generators/migration')
+    tmp.rmtree if tmp.exist?
+    tmp.mkpath
+
+    Dir.chdir(tmp)
+    @root = tmp
+  end
+
+  def chdir_to_root
+    Dir.chdir(@pwd)
+  end
+
+  before do
+    create_temporary_dir
+  end
+
+  after do
+    chdir_to_root
+  end
+
+  describe 'start' do
```

Comment:
How about following convention `#` prefix to denote a method? That is `#start` instead of `start`